airflow-core/src/airflow/ui/src/router.tsx

```diff
@@ -208,6 +257,15 @@ export const routerConfig = [
           { element: <TaskOverview />, index: true },
           { element: <TaskInstances />, path: "task_instances" },
           { element: <Events />, path: "events" },
+          {
+            // The following iframe sandbox setting is intentionally less restrictive.
+            // This is considered safe because the framed content originates from the Plugins,
+            // which is part of the deployment of Airflow and trusted as per our security policy.
+            // https://airflow.apache.org/docs/apache-airflow/stable/security/security_model.html
+            // They are not user provided plugins.
+            element: <Iframe sandbox="allow-scripts allow-same-origin allow-forms" />,
+            path: "plugin/:page",
+          },
```

Comment:
Maybe extract this into a really small fn. "getPluginIframeRoute" or something, in the same file.

Just so the comment, and how to construct the iframe component remains in one place and not copy pasted everywhere. I'm scared of someone updating in one place and not in other occurrences. (That doens't cost much to do and would help preventing mistakes in the future)